README.md

```diff
@@ -3,7 +3,7 @@
 ## Self Introduction
 My name is Yuwen Xiong. I am a second-year master student at **UChicago Harris School of Public Policy**. Below is the photo of mine:
 
-![Photo of Yuwen](C:\Users\Xyw\Documents\GitHub\CFSS\hw01\LinkedIn-profile-smaller.jpg)
+![Photo of Yuwen](LinkedIn-profile-smaller.jpg)
 
 If you want to know more about me, you can take a look at [my LinkedIn profile](https://www.linkedin.com/in/yuwenxiong/) .
```

Comment:
I'm going to be honest - you did a good job on this assignment, so I'm not really sure what notes to leave.  I don't have a LinkedIn, so I am unable to view your profile, but the link took me to the website so I'll assume it's there.

Reply:
LinkedIn is a good way to make people know you! So maybe start playing with it!